api/cases/case.js

```diff
@@ -11,10 +12,32 @@ function getCaseEvents(caseId, userId, options, caseType = 'Benefit', jurisdicti
     return generateRequest(`${config.services.ccd_data_api}/caseworkers/${userId}/jurisdictions/${jurisdiction}/case-types/${caseType}/cases/${caseId}/events`, options)
 }
 
-function getCaseWithEvents(caseId, userId, options, caseType = 'Benefit', jurisdiction = 'SSCS') {
+function postHearing(caseId, userId, options, jurisdictionId = 'SSCS') {
```

Comment:
Could we put these functions into a seperate directory/module?

Reply:
Just seen these functions DO exist in a seperate routes module and guessing these are maybe a copy?